src/codeu/chat/common/NetworkCode.java

```diff
@@ -49,5 +49,7 @@
       USER_STATUS_UPDATE_REQUEST = 41,
       USER_STATUS_UPDATE_RESPONSE = 42,
       CONVERSATION_STATUS_UPDATE_REQUEST = 43,
-      CONVERSATION_STATUS_UPDATE_RESPONSE = 44;
+      CONVERSATION_STATUS_UPDATE_RESPONSE = 44,
+      CHANGE_USER_PERMISSION_LEVEL_REQUEST = 46,
```

Comment:
You mentioned this one earlier, this should be 45 instead of 46 and the next one should be 47. This doesn't really matter since it's going to get fixed during the merge conflict resolution.